php/commands/comment.php

```diff
@@ -247,10 +247,16 @@ public function list_( $_, $assoc_args ) {
 	 */
 	public function delete( $args, $assoc_args ) {
 		parent::_delete( $args, $assoc_args, function ( $comment_id, $assoc_args ) {
-			$r = wp_delete_comment( $comment_id, \WP_CLI\Utils\get_flag_value( $assoc_args, 'force' ) );
+			$force = \WP_CLI\Utils\get_flag_value( $assoc_args, 'force' );
+
+			$r = wp_delete_comment( $comment_id, $force );
```

Comment:
What's the behavior if you call `wp_delete_comment()` on an already-trashed comment?